docs/Apple/release-notes.md

```diff
@@ -6,13 +6,108 @@ hide:
+
+- New DEX Telemetry attribute for iOS: **Percent Asleep**
+   - Name: percent_asleep
+   - Represents the duration percentage that the device has been asleep since it was booted (on a scale from 0 to 100).
+   
+- New DEX Telemetry attribute for iOS: **Percent Awake**
+   - Name: percent_awake
+   - Represents the duration percentage that the device has been awake since it was booted (on a scale from 0 to 100).
+
+### Resolved Issues
+
+- Creation of the automatic `WS1IntelligenceSDK-Enable-Completed` breadcrumb has been changed to an asynchronous operation to prevent a minor delay in completing the enable process.
+- The IntelSDK now attempts to connect to the `workspaceone.com` domain URLs instead of the previous `vmwservices.com` domain URLs. If the `workspaceone.com` domain URLs are accessible, IntelSDK will construct all its URLs using `workspaceone.com`.
+   - Requirements for allowlisting the necessary URLs can be found here:  [Requirements](https://developer.apple.com/documentation/webkit/wknavigationdelegate/3223382-webview)
+   - For more information on the domain migration, see: [Knowledge Base](https://kb.omnissa.com/s/article/6000882)
+
+- The dSYM upload feature will no longer be supported by the servers after `Jun 30, 2025`
+   - Please remove any build phases that were executing the script which uploaded dSYMs. After Jun 30, 2025, the upload attempt will fail with an error. 
+      - Errors will be prefixed with: `Symbol File upload failed with result`
+   - Additionally, delete the corresponding service account JSON file.
+   - For more information, see our KB article: [End of Availability & Support for the App Crash Symbolication](https://kb.omnissa.com/s/article/6000838)
+
+### Known Issues
+
+none
+
+
+## Omnissa IntelligenceSDK for iOS 25.1.2 Release - February 2025
+
+### Minimum Requirements
+
+- iOS 16.0 device or iPadOS 16.0 device
+- tvOS devices and app extensions are no longer supported
+- visionOS for Vision Pro devices is not supported
+
+### New Features
+
+none 
+
+### Resolved Issues
+
+- Fixes a potential deadlock.
+
+### Known Issues
+
+none
+
+
+## Omnissa IntelligenceSDK for iOS 25.1.1 Release - May 28, 2025
+
+### Minimum Requirements
+
+- iOS 16.0 device or iPadOS 16.0 device
+- tvOS devices and app extensions are no longer supported
+- visionOS for Vision Pro devices is not supported
+
+### New Features
+
+none 
+
+### Resolved Issues
+
+- The IntelSDK now attempts to connect to the `workspaceone.com` domain URLs instead of the previous `vmwservices.com` domain URLs. If the `workspaceone.com` domain URLs are accessible, IntelSDK will construct all its URLs using `workspaceone.com`.
+   - Requirements for allowlisting the necessary URLs can be found here:  [Requirements](https://developer.apple.com/documentation/webkit/wknavigationdelegate/3223382-webview)
```

Comment:
Can we update the link around allow-listing the required URL's to this - https://docs.omnissa.com/bundle/Intelligence/page/IntelRequirements.html ?

Reply:
Also updated the same bad link for the 25.4.0 release notes